feedback/tests/conftest.py

```diff
@@ -1,12 +1,15 @@
 import json
 import pickle
 from copy import deepcopy
+from unittest import mock
 
 import pytest
-from tweepy import API
+import responses
+from tweepy import API, OAuthHandler
 from tweepy.models import DirectMessage as TweepyDirectMessage
 
-from feedback.tests.data.direct_message_json import test_direct_message_json
+from feedback.custom_tweepy_api import custom_parser, CustomTweepyAPI
+from feedback.tests.data.direct_message_json import test_direct_message_json, new_direct_message_api_response
```

Comment:
I think these are in wrong order for `isort`, I guess you haven't run it?

Reply:
Oh, true! Included isort in Travis setup now.